src/router/index.js

```diff
@@ -71,7 +71,46 @@ const routes = [
 
 const router = createRouter({
   history: createWebHashHistory(),
+  base: process.env.FRONTEND,
   routes,
 });
 
+/*
+Router auth logic start
+
+Commented code below that works on `isAuthenticated` state and before every route:
+1. Redirects user to home if user is already logged in and visiting a page that is intended for guest (route.meta.guest)
+2. Redirects user to login if user is not authenticated and visits a page that requires authentication (route.meta.requiresAuth)
+
+Uncomment the lines below when the existing login checks that uses local storage are disabled.
+*/
+
+// router.beforeEach((to, from, next) => {
+//     if (to.matched.some((record) => record.meta.requiresAuth)) {
+//         if (store.getters.isAuthenticated) {
+//             next();
+//             return;
+//         }
+//         next("/login");
+//     } else {
+//         next();
+//     }
+// });
```

Comment:
this is great 👍 
does this mean we can remove the manual redirections to the login page that we have put in every page?

Reply:
Yep!

Reply:
Fixed in #142